hscontrol/app.go

```diff
@@ -1012,3 +1011,46 @@ func readOrCreatePrivateKey(path string) (*key.MachinePrivate, error) {
 
 	return &machineKey, nil
 }
+
+func (h *Headscale) loadACLPolicy() error {
+	var (
```

Comment:
We should probably have a fatal if `acl_policy_path` is set so people are forced to migrate.
Alternative we could have a warn, and if it is set we default to:

```yaml
policy:
   mode: file
   path: <Same as `acl_policy_path`>
```

Reply:
Thanks for adding the required piece of code. 🙇🏼